src/content/ui-api/kubectl-gs/template-app.md

```diff
@@ -63,6 +63,7 @@ spec:
 
 It also supports the following optional flags:
 
+- `--app-name`: Name of the App CR. Can be set when you want to change the name or install multiple instances of an app.
```

Comment:
Can we also mention here where the default value is coming from, please?

Reply:
Yes thanks I've added.